what_is.md

```diff
@@ -15,16 +15,189 @@ kernelspec:
+stored once. As the trees get larger and larger, the efficency
+gains become significant. For instance, if we simulate the evolution of a 10 billion
+human chromosomes - roughly every living human - and store the resulting DNA sequences in
+tree sequence format, it take 20,000 times less space than using the conventional
+(so-called VCF) format and is about XXX times faster to process.
+
 (sec_what_is_ancestry)=
 
 ## A record of genetic ancestry
 
-(sec_what_is_dna_data)=
+Often, it turns out that what we are interested in is not the DNA sequence data, but the
+genealogy itself. Knowledge of the ancestry can be used, for instance, to determine the
+origin and age of variants under selection, to capture the spatial structure of
+populations, or to uncover the effects of hybridization and admixture in the past.
+Tree sequences are a particularly powerful representation because of how
+closely they are linked to the underlying biological processes that generated the genomes
+in the first place.
+
+For example, each branch point in one of the trees above represents a most recent common
+ancestor (MRCA), in other words a genome which existed at a specific time in the past.
+It is helpful to distinguish these *ancestral* genomes from the *sampled* genomes
+($\mathrm{a}$ to $\mathrm{j}$) which we have measured more directly. We can
+indicate this in our trees by adding the MRCA genomes as circular nodes, rather than the
+squares we have used for sampled genomes. 
+
+```{note}
+For clarity in these examples, we have been relabelling the sample genomes as
+$\mathrm{a}$ to $\mathrm{j}$, and we will continue to do so. However, in standard tree
+sequences, all the genomes (which are referred to as "nodes"), including the samples,
+are numbered sequentially from 0. The ancestral nodes are in this example are therefore
+labelled from 10 upwards.
+```
+
+```{code-cell} ipython3
+all_labels = {n:n for n in range(ts.num_nodes)}
+all_labels.update(labels)  # For consistency, keep letters for the sample nodes
+tree_nodes = [set(tree.nodes()) for tree in ts.trees()]
+kept_nodes = set.intersection(*tree_nodes)
+changed_nodes = set(range(ts.num_nodes)) - kept_nodes
+style2 = ",".join(f"#svg2 .n{n} > .sym" for n in changed_nodes) + "{fill:red}"
+style2 += "#svg2 .node > .sym {visibility: visible;}"  # force-show all nodes: not normally needed
+SVG(ts.draw_svg(size=sz, root_svg_attributes={'id':'svg2'}, node_labels=all_labels, style=style2))
+```
 
-## An encoding of DNA data
+Here we have also coloured those nodes (genomes) that only exist in some of the trees.
+With longer chromosomes or higher recombination rates, essentially all the ancestral
+nodes will fall into this category. Biologically, this means that most ancestral nodes
+represent only a fraction of the whole ancestral genome that must have existed in the past.
 
+The change from one tree to another is also biologically meaningful. It indicates that
```

Comment:
I think it would be nice to hand off to the ARG tutorial somewhere, for people like Jotun's student who don't realise that tree sequences essentially *are* ARGs. We should use the words "Ancestral Recombination Graph" somewhere on the "what is" page, both for search engines and for (the tiny minority of) people who are specifically hunting for this term: after all, one answer to the "what is a tree sequence?" question is that it's basically an expanded representation of an ARG. But this can be done as a throw-away comment with a link, possibly somewhere else, such as at the end of the page.